docs/types/mapping-types.rst

```diff
@@ -30,7 +30,13 @@ You can mark state variables of mapping type as ``public`` and Solidity creates
 parameter for the getter. If ``_ValueType`` is a value type or a struct,
 the getter returns ``_ValueType``.
 If ``_ValueType`` is an array or a mapping, the getter has one parameter for
```

Comment:
This doesn't really make sense. Which `_KeyType`? The getter has additional parameters as needed to access the final element in the mapping / array structure.

Reply:
@chriseth it refers to the paragraph a couple above that uses the same analogies, but I'll make the connection between the two paragraphs clearer.

Reply:
well, the keytype is mentioned in line 7. But for clarification we could write "the mappings keytype" / "the keytype of the mapping"
On that note, maybe a cod example is useful here, too

Reply:
There is a code example a paragraph later, might be better to view the overall changes to see everything in context and let me know if you think anything is still needed.

Reply:
Ah sorry, now I see it!